src/components/EditableTableCell/EditableTableCell.tsx

```diff
@@ -70,9 +71,9 @@ export const EditableTableCell: React.FC<EditableTableCellProps> = props => {
   return (
     <TableCell className={clsx(classes.container, className)}>
       {opened && <div className={classes.overlay} onClick={disable} />}
-      <Typography variant="caption" onClick={enable} className={classes.text}>
+      <Text size={2} fontWeight="light" onClick={enable} className={classes.text}>
```

Comment:
question: Didn't you lost color here?

Reply:
I (surprisingly) didn't. Current implementation of Material theme makes caption have the same colour as the rest